generators/info/index.js

```diff
@@ -52,25 +52,33 @@ module.exports = ReportGenerator.extend({
             let result = shelljs.cat('.yo-rc.json');
             result = result.replace(/"rememberMeKey": ".*"/g, '"rememberMeKey": "replaced-by-jhipster-info"');
             result = result.replace(/"jwtSecretKey": ".*"/g, '"jwtSecretKey": "replaced-by-jhipster-info"');
-            this.log(`${'\n##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**\n' +
-                '\n```yaml\n'}${result}\n\`\`\`\n`);
+            console.log(`${'\n##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**\n' +
+                '\n<details>\n<summary>.yo-rc.json file</summary>\n<pre>\n'}${result}\n</pre>\n</details>`);
             done();
         },
 
         displayEntities() {
             const done = this.async();
-            this.log('\n##### **JDL for the Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**\n');
-            const jdl = this.generateJDLFromEntities();
-            this.log(`\n\`\`\`yaml\n${jdl.toString()}\n\`\`\`\n`);
+            console.log('\n##### **JDL for the Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**\n');
+            try {
+                this.getExistingEntities().forEach((entity) => {
```

Comment:
Is there a way to build a link with the JDL definition so that we can see the visual representation?